docs/request-sequence.svg

```diff
@@ -0,0 +1,6 @@
+
+:root { --mermaid-font-family: "trebuchet ms", verdana, arial;}</style><style>#mermaid-1587532740696 {
```

Comment:
For some reason this image won't preview and the `</style>` on this line is highlighted red

Reply:
You can see the file renders correctly here (by navigating to the file and selecting to view 'raw'): https://raw.githubusercontent.com/ServiceInnovationLab/korerorero-reverse-proxy/87b711bfbf41440ceee500886b8c94273c3c084b/docs/request-sequence.svg

I'm going to put this down to a githubism with a complex SVG.